src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -156,32 +158,49 @@ private async Task StartHostAsync(CancellationToken cancellationToken, int attem
             }
             catch (OperationCanceledException)
             {
+                GetHostLogger(localHost).LogDebug("Host startup was canceled.");
                 throw;
             }
             catch (Exception exc)
             {
-                LastError = exc;
-                State = ScriptHostState.Error;
-                attemptCount++;
+                // Another host has been created before this host
```

Comment:
doesn't this text belong in the else block below?

Reply:
Yep -- missed while refactoring. Will move.